SmellDetector/SmellModel/CaDETClassDTO.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using SmellDetector.Controllers;
+
+namespace SmellDetector.SmellModel
+{
+    public class CaDETClassDTO
+    {
+        public Dictionary<string, MetricsDTO> IdentifierAnalyses { get; set; }
```

Comment:
Still a pretty mysterious name "Analize identifikatora". Metric calculations aren't really analyses, and its unclear what identifier refers to.

We need to think about what we are receiving from the RepoCompiler https://github.com/Clean-CaDET/platform-documentation where the answer is metrics and features for various code items. Since we currently have only metrics, we can rename this field to something like CodeItemMetrics.